libparsec/src/cli/list_devices.rs

```diff
@@ -0,0 +1,20 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+use std::path::Path;
+
+use crate::{cli::*, list_available_devices};
+
+pub async fn list_devices(config_dir: &Path) {
+    let devices = list_available_devices(config_dir).await;
+    let n = devices.len();
+    let slug_len = 2 + (n + 1).ilog2() as usize;
```

Comment:
Why we limit the len of the slug by the number of devices ?

Reply:
I forgot the comment there

Reply:
Done